packages/strapi-plugin-content-manager/admin/src/components/InputUID/index.js

```diff
@@ -220,7 +227,7 @@ const InputUID = ({
                   <RegenerateButton
                     onMouseEnter={handleGenerateMouseEnter}
                     onMouseLeave={handleGenerateMouseLeave}
-                    onClick={generateUid.current}
+                    onClick={() => generateUid.current()}
```

Comment:
Why?

Reply:
Because `onClick` pass an event to the function so the shouldSetInitialValue will be true every time. We don’t wan’t this one to be true here.